presentation/common/src/presentation-common/Utils.ts

```diff
@@ -86,6 +86,7 @@ export const getInstancesCount = (keys: Readonly<KeySet>): number => {
 export const DEFAULT_KEYS_BATCH_SIZE = 5000;
 
 /** @internal */
+declare const __dirname: string; // eslint-disable-line @typescript-eslint/naming-convention
```

Comment:
Is this sufficient? I think this won't work when compiled as an es module.

Reply:
The purpose here is for the code to execute in the browser without a runtime error. To properly use __dirName in node with esm, we would probably need something like this:

const __dirname = new URL(".", import.meta.url).pathname;

However, that assumes we are full on es modules. I think much more work needs to be done before we get there.

Reply:
probably the proper fix here would be to add it to window based on the process detector. I will update

Reply:
@grigasp updated the code

Reply:
> The purpose here is for the code to execute in the browser without a runtime error.

I think the purpose should be to get a valid path to the assets. Setting to an empty string avoids a runtime error, but I doubt it will help us find the assets.

I'm also not a fan of assigning global variables like this. Instead, I think this is better:
```ts
export const PRESENTATION_COMMON_ROOT = window.__dirname ?? "";
```
Of course it doesn't solve the invalid assets path problem.

Reply:
My take on this code is that it was added specifically with node (electron/native) in mind and not mindful of the browser, and thus the browser should not care about it (for now). __dirName alone would not have had a value in the browser but it at least would not cause an error when building with Webpack due to its node polyfills. We are attempting to move away from Webpack and thus the necessity of a change. 

The problem with your suggestion is that it would not work in node (which is where it currently does work and where we care about it working, if I am correct). Perhaps I'm incorrect about this?